app/jobs/batch_ingest_job.rb

```diff
@@ -25,13 +42,13 @@ def perform(filename)
     ingest = Avalon::Batch::Ingest.new(collection)
     begin
       package = Avalon::Batch::Package.new(filename, collection)
-      ingest.ingest_package(package)
+      ingest.process_valid_package(package: package)
```

Comment:
Is there a reason why you are skipping the package validation here?  Wouldn't you still want to do that or is the lambda taking care of that part?

Reply:
I'm okay with the lambda if it gets included in the cloud formation in https://github.com/avalonmediasystem/avalon-aws

It would be nice to maybe make a note to that effect above.  Maybe in line 17 "an AWS Lambda, which runs validations then queues the job below."?